Firestore/core/src/firebase/firestore/local/leveldb_key.h

```diff
@@ -416,6 +416,18 @@ class LevelDbDocumentTargetKey {
    */
   static std::string SentinelKey(const model::DocumentKey& document_key);
 
+  /**
+   * Given a sequence number, encodes it for storage in a sentinel row.
+   */
+  static std::string EncodeSentinel(
+      const model::ListenSequenceNumber sequence_number);
+
+  /**
+   * Given an encoded sentinel row, return the sequence number.
+   */
+  static model::ListenSequenceNumber DecodeSentinel(
+      const absl::string_view& slice);
```

Comment:
`absl::string_view` is intended to be passed by value (it's just a pointer and a size, which is cheaper to just copy rather than take a reference).

Reply:
Done.